_docs/guides/integrating-vms.md

```diff
@@ -31,34 +31,41 @@ this infrastructure as a single mesh.
   [Installation guide]({{home}}/docs/setup/kubernetes/quick-start.html).
 
 * Deploy the [BookInfo]({{home}}/docs/guides/bookinfo.html) sample application.
-
-* Create a VM named 'db' in the same project as Istio cluster, and [Join the Mesh]({{home}}/docs/setup/kubernetes/mesh-expansion.html).
+* Create a VM named 'mysqldb' in the same project as Istio cluster, and [Join the Mesh]({{home}}/docs/setup/kubernetes/mesh-expansion.html).
```

Comment:
in the same project as your Istio cluster
(or the Istio cluster) ?

Reply:
also the name of the VM doesn't (should not) matter; maybe "vm-1" is better to not imply otherwise ?